Sources/Observers/KeyboardObserver.swift

```diff
@@ -13,6 +13,10 @@ public final class KeyboardObserver: NSObject {
         }
     }
 
+    /// List of view subclasses that should be ignored by the gesture recognizer. Touches on these views won't get
+    /// the keyboard resigned. Default is `[UIControl.self]`.
+    public var ignoredViews: [UIView.Type] = [UIControl.self]
+
     public init(window: UIWindow, shouldTapCancelTouches: Bool = false) {
```

Comment:
we could inject the ignoredViews though the init with default value, no? 💄 

```swift
public var ignoredViews: [UIView.Type]

public init(window: UIWindow, shouldTapCancelTouches: Bool = false, ignoredViews: [UIView.Type] = [UIControl.self]) {
    ...
}
```

Reply:
Done.